datahub-web-react/src/app/permissions/policy/policyUtils.ts

```diff
@@ -126,3 +127,150 @@ export const setFieldValues = (
     }
     return { ...filter, criteria: [...restCriteria, createCriterion(resourceFieldType, fieldValues)] };
 };
+
+
+/**
+ * Add an entry to the ListPolicies cache.
+ */
+export const addToListPoliciesCache = (client, newPolicy, pageSize, query) => {
+    // Read the data from our cache for this query.
+    const currData: ListPoliciesQuery = client.readQuery({
+        query: ListPoliciesDocument,
+        variables: {
+            input: {
+                start: 0,
+                count: pageSize,
+                query,
+            },
+        },
+    });
+
+    const completeNewPolicy = {
+        ...newPolicy,
+        actors : {
```

Comment:
actors: newPolicy.actors ? newPolicy.actors : null,
resources: newPolicy.resources ? newPolicy.resources : null

Reply:
The thing to keep in mind:

Apollo cache will throw / fail if it requests a field and there is no value for it, EVEN IF THE FIELD CAN BE NULL.

NULL has to be explicitly returned if the field is requested.

List policies query:

.... 

start
count
total 
policies {
    resources {

   }
   actors {

   }
}

Reply:
Use apollo chrome extension to help debug